build

```diff
@@ -124,4 +124,9 @@ if [ -d cert ]; then
 	container_mount_opts+=(-v "$PWD/cert:/builder/cert:ro")
 fi
 
+if [ "$container_engine" = "docker" ] && out=$(sysctl kernel.apparmor_restrict_unprivileged_userns 2> /dev/null) && [[ $out = "kernel.apparmor_restrict_unprivileged_userns = 1" ]]; then
+	echo Allow unprivileged user namespaces, use podman to avoid this setting
```

Comment:
How is unprivileged userns restriction via apparmor related to podman or docker? shouldn't both face the same issue if unprivileged userns are disabled?